Makefile

```
@@ -349,3 +349,27 @@ integration-test:
 	else\
 		$(GINKGO) -v $(TEST_ARGS) --fail-fast --focus-file=$(TEST_FILE) test/integration;\
 	fi
+
+#hosted
+
+.PHONY: bootstrap-hosted
+bootstrap-hosted: clone-hosted install-hosted copy-config install-crds install-resources kind-deploy-policy-framework kind-deploy-policy-controllers
+
+.PHONY: install-hosted
+install-hosted: 
+	@cd ./governance-policy-addon-controller && KIND_VERSION=latest HOSTED_MODE=true ./build/manage-clusters.sh
+
+.PHONY: 
+copy-config:
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig ./kubeconfig_hub
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig-internal ./kubeconfig_hub_internal
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig ./kubeconfig_managed
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig-internal ./kubeconfig_managed_internal
+
+.PHONY: clone-hosted
+clone-hosted:
+	-git clone --depth=1 https://github.com/open-cluster-management-io/governance-policy-addon-controller.git
```

Could you please use the stolostron version since these tests aren't run in the upstream community yet?
```suggestion
	-git clone --depth=1 https://github.com/stolostron/governance-policy-addon-controller.git
```